lib/agent0/agent0/hyperdrive/exec/execute_agent_trades.py

```diff
@@ -294,7 +291,7 @@ async def async_match_contract_call_to_trade(
                     amount=trade_result.base_amount,
                     unit=TokenType.BASE,
                 ),
-                longs={trade.mint_time: Long(-trade_result.bond_amount)},
+                longs={trade.maturity_time: Long(-trade_result.bond_amount)},
```

Comment:
I'm confused about whether this is an `int` or not -- there is a TODO elsewhere saying it should be an `int`, but it's a fixed point. In the hyperdrive wallet typedef it asks for `int`. A few lines back (264) in this file there is a new variable `maturity_time_seconds = int(trade.maturity_time)`, implying it is not an int.